r_code/synthetic_dgp.R

```diff
@@ -596,13 +586,13 @@ NoisifyDraw <- function(data_inp, seed, log_output = T, sig_y = 0.2) {
 #' @param cov_overlap_inp Number between (-1,1) shifting distribution of
 #'    covariates for a random fraction (prop_treated) of generated x variables.
 #'    A value of 1 shifts the distribution for treatment up on all x's,
-#'     whereas -1 shifts up the distribution for donors.
+#'    whereas -1 shifts up the distribution for donors.
 #' @param loading_scale_inp Number in (-1,1) shifting loading distribution by
 #'    treatment. Negative values shift loadings up for control units, positive
-#'     values shift loadings distribution up for treated units.
+#'    values shift loadings distribution up for treated units.
 #' @param num_factors_inp Integer number of time-varying, unobserved factors to
 #'    simulate. For freq="monthly", "weekly", "daily", number of factors must be
-#'     at least 3,4,and 5 respectively.
+#'    at least 3,4,and 5 respectively.
```

Comment:
```suggestion
#'    at least 3, 4, and 5 respectively.
```

Please change all instances in the code.